backend/modules/common/domain/exceptions.py

```diff
@@ -4,3 +4,7 @@ class DataValidationException(Exception):
 
 class FileDataFormatNotSupportedException(Exception):
     pass
+
+
+class DataLoadException(Exception):
```

Comment:
It's too generic.
I would simple name it when it's raised.

From my perspective, it's supposed to be named when saving darta failed, so I would go with

`DataSaveException`